### DISCUSSION TITLE
Using GitHub Templates for octoherd 'script' repositories

### DISCUSSION THREAD

**Discussion by oscard0m** [octoherd/octoherd#8] [Ideas]
Right now there is a script utility to create an 'octoherd' script repository locally and push it remotely under user's GitHub account.

Would have sense to add the option, for maintainers of this project, to create an octoherd script from a repository template? If it has sense, maybe could have sense to discuss a workflow for external contributors publishing their scripts under [Octoherd Project](https://github.com/octoherd?type=source) ?

**Comment by gr2m** [comment]
Just to make sure I understand, you are asking two things

1. Can we create a template repository that script developers could use as a starting point to create a new script repository?
2. Can external script developers create script repositories within the @octoherd organization?

Is that correct?

For `1.`, I think it's a good idea, but I do want to create an init script as I described in yesterday's project update:
https://github.com/octoherd/octoherd/discussions/6#discussioncomment-424502. An init script can go beyond just creating a repository

For `2.`, I don't think that's necessary. In order to keep track of scripts I'd ask script developers to do two things

1. Add the `octoherd-script` topic to the repository
2. Install the @octoherd app - (yet to be done)

We could use either as a source of all Octoherd Scripts out there, even when they are spread across multiple GitHub user and organization accounts

**Reply by oscard0m** [reply]
You understood what I meant and... WOW! You are always 2 steps ahead! I love the idea
